### DISCUSSION TITLE
Fine tuning CHGNet with DFT energies and forces

### DISCUSSION THREAD

**Discussion by Atefeh-Yadegarifard** [CederGroupHub/chgnet#62] [Q&A] [answered]
Dear CHGNet developers,
Hello. 
I would like to request an example for how CHGNet can be trained with additional DFT data (calculated by the users). Specifically the format of the data that has to be used for training. 
Thank you.

**Comment by bowen-bd** [comment, marked as answer]
For fine-tuning code, please refer to:
https://github.com/CederGroupHub/chgnet/blob/main/examples/fine_tuning.ipynb

For parsing VASP output into fine-tuning dataset, please refer to:
https://github.com/CederGroupHub/chgnet/blob/main/chgnet/utils/vasp_utils.py

**Reply by nckien84** [reply]
@bowen-bd : It seems [vasp_utils.py](https://github.com/CederGroupHub/chgnet/blob/main/chgnet/utils/vasp_utils.py) only support a single DFT calculations. Is there any way to find all DFT runs and convert it to single JSON file for later fine-tuning process?
